docs/user_guide/hopsworks/jupyter.rst

```diff
@@ -32,6 +32,23 @@ and then `Help-Launch Classic Notebook`.
 
     Start a Jupyter notebook server
 
+**Using the previous attached configuration**
+
+When you run a notebook, the jupyter configuration used is stored and attached to the notebook as xattribute. You
+can use this configuration later to start the jupyter notebook server directly from the notebook file. In the
+jupyter folder, when the notebook you previously run is selected, you will see options to view the previously run
```

Comment:
You can remove the "In the jupyter folder," part. It should be enough to phrase it "When the notebook you previously run is selected..."